apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -137,6 +151,59 @@ const AppearanceView = ({
   return (
     <div>
       <Meta title={t("appearance")} description={t("appearance_description")} borderInShellHeader={false} />
+      <div className="border-subtle mt-6 flex items-center rounded-t-lg border p-6 text-sm">
+        <div>
+          {/* TODO: support i18n: we should support other languages in other PR */}
+          <p className="text-default text-base font-semibold">{t("app_theme")}</p>
+          <p className="text-default">{t("app_theme_applies_note")}</p>
```

Comment:
we should use the text mentioned in the issue comment by peer

https://github.com/calcom/cal.com/issues/11997#issuecomment-1828153608

"Dashboard theme"

Reply:
Got it! I've modified the text. I should read the issue more carefully next time. Thanks!